picard/tagger.py

```diff
@@ -287,9 +301,9 @@ def _run_init(self):
                     self.add_directory(decode_filename(file))
                 else:
                     files.append(decode_filename(file))
-            if files:
-                self.add_files(files)
             del self._cmdline_files
+            if files:
+                self.add_files([files])
```

Comment:
P.S. About to rename tagger.py#add_files to _add_files and create a stub add_files to handle the Add Files dialog which @samj1912 picked up on.